Northstar.CustomServers/mod/scripts/vscripts/mp/_stats.nut

```diff
@@ -136,12 +136,12 @@ float function PlayerStat_GetCurrentFloat( entity player, string statCategory, s
 	return 0
 }
 
-void function UpdatePlayerStat( entity player, string statCategory, string subStat, int count = 1 )
+void function UpdatePlayerStat( entity player, string statCategory, string subStat, int count = 1, string alias = "" )
```

Comment:
```suggestion
void function UpdatePlayerStat( entity player, string statCategory, string subStat, int count = 1, string statAlias = "" )
```
Name could be better maybe

Also, i would prefer if this new optional arg came before count, i cant remember if anything uses the other optional arg atm. If it would require other changes to do this, don't bother

Reply:
Couple codes uses the count argument, but they just specify this default 1, not sure if that is enough to consider not swapping the order of the arguments.

Reply:
Yeah, don't bother then